### PR TITLE
V1.1.3-hotfix1

### PR DESCRIPTION
**Pull request by cvadityaflipkart**
https://github.com/SpectoLabs/hoverfly/issues/895

### CODE REVIEW

File: core/matching/matchers/json_path_match.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func JsonPathExecution(matchString, toMatch string) (string, error) {
 		return "", err
 	}
 
-	var data map[string]interface{}
+	var data interface{}
```
Comment (tommysitu) [inline core/matching/matchers/json_path_match.go]: 👍

File: core/matching/matchers/json_partial_match.go [hunk 2]
```diff
@@ -7,22 +7,43 @@ import (
...
+		}
+		nodes := getAllNodesFromArray(actual)
+		for _, node := range nodes {
+			if mapContainsMap(expected, node) {
+				return true
+			}
+		}
+	default:
+		var actual map[string]interface{}
+		err3 := json.Unmarshal([]byte(toMatch), &actual) // has type interface{}
```
Comment (tommysitu) [inline core/matching/matchers/json_partial_match.go]: The json is already unmarshaled to `toMatchType`, you can do a type conversion here:
```
actual, ok := toMatchType.(map[string]interface{})
if !ok {
        // log an error
	return false
}
```

Comment (cvadityaflipkart) [reply core/matching/matchers/json_partial_match.go]: Done

**Comment by cvadityaflipkart** [comment]
@tommysitu Welcome!!! Will this be released as a hotfix for v1.1.3 ?

**Comment by tommysitu** [comment]
Thanks! will check it out today. I'm planning to release it in  v1.1.4 this week.

File: core/matching/matchers/json_partial_match.go [hunk 3]
```diff
@@ -7,24 +7,37 @@ import (
...
 	if err0 != nil || err1 != nil {
 		return false
 	}
 
-	nodes := getAllNodesFromMap(actual)
-	for _, node := range nodes {
-		if mapContainsMap(expected, node) {
-			return true
+	actual, ok := toMatchType.(map[string]interface{})
+	if !ok {
```
Comment (tommysitu) [inline core/matching/matchers/json_partial_match.go]: let's do ` if ok .... else ...` rather then the opposite

Comment (tommysitu) [reply core/matching/matchers/json_partial_match.go]: I will do that on my end

**Comment by tommysitu** [comment]
Thanks for the fix! @cvadityaflipkart A release should be out sometime today

**Comment by cvadityaflipkart** [comment]
@tommysitu Welcome

**Pull request merged**